docker/k8s/vtadmin/Dockerfile

```diff
@@ -43,8 +43,13 @@ RUN deluser node && \
    groupadd -r --gid 2000 vitess && \
    useradd -r -g vitess --uid 1000 vitess && \
    chown -R vitess:vitess /vt && \
-   chown -R vitess:vitess /vtdataroot
+   chown -R vitess:vitess /vtdataroot && \
+   # the entrypoint need to edit vtadmin configuration, so it needs to be writable by every user
+   chmod 777 /vt/web/vtadmin/build/index.html
```

Comment:
I don't understand why that needs to be writeable. Changes aren't persisted to the file, they are generally stored in the topo

Reply:
Hmm... Shouldn't making it writable by the vitess user be sufficient like the other folders? IIUC, the entrypoint should be executed as that user

Reply:
no because we want to allow user to run the image with any user (like in the vitess operator with an non existing one)